vertical-pod-autoscaler/examples/hamster-vpa.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: "poc.autoscaling.k8s.io/v1alpha1"
```

Comment:
I think hamster-deployment and hamster-vpa might be a single yaml file, unless you really want to use them separately which I doubt.

Reply:
Done.